### ISSUE TITLE
Avoiding the dynamic Value type

### ISSUE THREAD

**Issue by jneem** [little-dude/rmp-rpc#13]
It would be nice to have an API that doesn't involve passing all of the msgpack data through dynamically allocated `rmpv::Value`s.

**Comment by ghost** [comment]
I agree. It's pretty awkward to use your own structs with this, because you have to work between `MyStruct <=> Vec<u8> <=> Value`.
